src/main/java/com/example/bankservicedemo/controller/AccountController.java

```diff
@@ -0,0 +1,41 @@
+package com.example.bankservicedemo.controller;
+
+import com.example.bankservicedemo.mapper.AccountMapper;
+import com.example.bankservicedemo.model.dto.account.AccountRequestDto;
+import com.example.bankservicedemo.model.dto.account.AccountResponseDto;
+import com.example.bankservicedemo.service.AccountService;
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/accounts")
+@AllArgsConstructor
+public class AccountController {
+    private final AccountService accountService;
+    private final AccountMapper accountMapper;
+
+    @PostMapping()
+    public void createAccount(AccountRequestDto accountRequestDto) {
+        accountService.save(accountMapper.toEntity(accountRequestDto));
+    }
+
+    @GetMapping("/{phone}")
```

Comment:
please see the task description and URL mappings that should be created
https://mate-academy.github.io/jv-program-fulltime/05_spring_boot/bank_service.html